xbmc/Util.cpp

```diff
@@ -1512,7 +1512,7 @@ bool CUtil::SupportsWriteFileOperations(const std::string& strPath)
     for (const auto& addon : CServiceBroker::GetVFSAddonCache().GetAddonInstances())
     {
       const auto& info = addon->GetProtocolInfo();
-      if (info.type == url.GetProtocol() && info.supportWrite)
+      if (info.type.find(url.GetProtocol()) != std::string::npos && info.supportWrite)
```

Comment:
This isn't correct. First split protocols and then search for an exact match.
e.g `"https".find("http") != std::string::npos` or `"sftp".find("ftp") != std::string::npos` would be true, but shouldn't.

Reply:
okay, split and find now.